lib/utils.js

```diff
@@ -50,7 +50,8 @@ function isDomElement(o) {
 }
 
 function isSpecialClick(event) {
-  return event.altKey || event.ctrlKey || event.metaKey || event.shiftKey;
+  let isMiddleClick = (event.button === 1);
```

Comment:
I feel it more readable, but this is mostly a matter of personnal preference. If it looks weird to you, I'm ok to change it. We just have to agree on when to put them or not :)